bob-backend/src/core.rs

```diff
@@ -445,38 +450,57 @@ impl Backend {
         self.inner.free_least_used_resources().await
     }
 
-    pub async fn delete(&self, key: BobKey) -> Result<u64, Error> {
+    pub async fn delete(
+        &self,
+        key: BobKey,
+        options: BobOptions,
+        force_delete: bool,
+    ) -> Result<(), Error> {
         let (vdisk_id, disk_path) = self.mapper.get_operation(key);
-        let mut ops = vec![];
-        if let Some(path) = disk_path {
-            ops.push(Operation::new_local(vdisk_id, path.clone()));
+        if !options.remote_nodes().is_empty() {
+            for node_name in options.remote_nodes() {
```

Comment:
This logic should be changed:
1. Get all nodes containing specific vdisk from `mapper`
2. Set `force_delete` only for nodes, that are in `options.force_alien_nodes` list
3. Call `delete_single` on all that nodes